ra/ra.go

```diff
@@ -1912,9 +1954,10 @@ func (ra *RegistrationAuthorityImpl) PerformValidation(
 		if prob != nil {
 			challenge.Status = core.StatusInvalid
 			challenge.Error = prob
-			go ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier.Value)
+			go ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier)
 		} else {
 			challenge.Status = core.StatusValid
+			ra.resetAccountPausingLimit(vaCtx, authz.RegistrationID, authz.Identifier)
```

Comment:
No need to do this synchronously, it can happen in the background. That's why it doesn't bother to return an error.
```suggestion
			go ra.resetAccountPausingLimit(vaCtx, authz.RegistrationID, authz.Identifier)
```